docs/vi/week01/01.md

```diff
@@ -0,0 +1,19 @@
+---
+Tham khảo: 01.md
+Tiêu đề: Tuần_01
```

Comment:
Same as above, please, please, check other languages and how they've translated.
@xcastilla can you help?

Reply:
Ok. I will correct it as soon as when I come back my office. I think I will translate all of them, so I was wrong when I translated this file.